Representor/HTTP/APIBlueprint/BlueprintTransition.swift

```diff
@@ -76,26 +76,33 @@ func parseAttributes(dataStructure:[String:AnyObject], builder:HTTPTransitionBui
 }
 
 extension HTTPTransition {
-  public static func from(# resource:Resource, action:Action, URL:String? = nil) -> HTTPTransition {
+  public static func from(resource  resource:Resource, action:Action, URL:String? = nil) -> HTTPTransition {
     return HTTPTransition(uri: URL ?? action.uriTemplate ?? resource.uriTemplate) { builder in
       builder.method = action.method
 
-      let addParameter = { (parameter:Parameter) -> Void in
+      func addParameter(parameter:Parameter) {
         let value = parameter.example
         let defaultValue = (parameter.defaultValue ?? nil) as NSObject?
         builder.addParameter(parameter.name, value:value, defaultValue:defaultValue, required:parameter.required)
       }
 
-      action.parameters.map(addParameter)
+      for parameter in action.parameters {
```

Comment:
Nope, but Swift 2.0 has introduced warnings for when you use `map` with side-effects (since I'm not using the result).